### PR TITLE
Update release-team membership to reflect all shadows

### PR DESCRIPTION
**Pull request by jeremyrickard**
This PR updates the release-team members to reflect the entire 1.20 team (we had missed a few shadows). I also updated some of the comments just for consistency.

/hold until reviewed by sig-release leads and RT sub team leads
/sig release

ref #kubernetes/sig-release#1306
cc: @kubernetes/release-engineering @kubernetes/release-team 

Signed-off-by: Jeremy Rickard <rickardj@vmware.com>

### CODE REVIEW

File: config/kubernetes/sig-release/teams.yaml
Comment (hasheddan) [inline config/kubernetes/sig-release/teams.yaml:278]: nitpick: others are updated to be prefixed with `1.20 RT`

**Review by cpanato — APPROVED** [review]
/lgtm

File: config/kubernetes/sig-release/teams.yaml
Comment (bai) [inline config/kubernetes/sig-release/teams.yaml:278]: nit:
```suggestion
        - sayanchowdhury # 1.20 RT Bug Triage Shadow
```

**Comment by bai** [comment]
Bug Triage changeset LGTM, thanks! 👍 

/lgtm
/approve

File: config/kubernetes/sig-release/teams.yaml
Comment (jeremyrickard) [reply config/kubernetes/sig-release/teams.yaml:278]: UGH I THOUGHT I GOT THEM ALL. HA thanks @hasheddan

**Review by justaugustus — CHANGES_REQUESTED** [review]
Nitty nit, but can you actually drop `RT` everywhere?
`1.20` in front is enough to denote that they're on the Release Team.

`RT` was used specifically to denote a Release Team Lead or Release Team Lead Shadow from the other roles.

Other thing to fix: The test that's failing is because @shubhamtatvamasi is not an org member. Please drop them from this PR and request they file a [request for org membership](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E).

**Comment by jeremyrickard** [comment]
> Nitty nit, but can you actually drop `RT` everywhere?
> `1.20` in front is enough to denote that they're on the Release Team.
> 
> `RT` was used specifically to denote a Release Team Lead or Release Team Lead Shadow from the other roles.
> 
> Other thing to fix: The test that's failing is because @ShubhamTatvamasi is not an org member. Please drop them from this PR and request they file a [request for org membership](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E).

Yep!

**Comment by jeremyrickard** [comment]
@justaugustus cleaned up the comments here (and in milestone maintainers for good measure) and cleaned up a bit.

**Comment by justaugustus** [comment]
Thanks @jeremyrickard!
/lgtm
/approve
/hold cancel

**Pull request merged**
